docs/en/deploy/Running-Alluxio-On-a-Cluster.md

```diff
@@ -217,6 +217,6 @@ and then [restart the service](#restart-alluxio).
 ### Update Worker-side Configuration
 
 If you only need to update some local configuration for a worker (e.g., change the mount
```

Comment:
Would it be "change the amount of storage" instead?

Reply:
You are right, I do not really understand this sentence at the beginning.